src/GafferUIModule/ViewportGadgetBinding.cpp

```diff
@@ -123,7 +129,7 @@ void GafferUIModule::bindViewportGadget()
 		.def( "gadgetToRasterSpace", &ViewportGadget::gadgetToRasterSpace, ( arg_( "gadgetPosition" ), arg_( "gadget" ) ) )
 		.def( "rasterToWorldSpace", &ViewportGadget::rasterToWorldSpace, ( arg_( "rasterPosition" ) ) )
 		.def( "worldToRasterSpace", &ViewportGadget::worldToRasterSpace, ( arg_( "worldPosition" ) ) )
-		.def( "render", &ViewportGadget::render )
+		.def( "render", &render, ( arg_( "currentStyle" ) = object() ) )
```

Comment:
The render method we're binding doesn't have a currentStyle argument, does it?

Reply:
Mh, I just blindly copied it from [here](https://github.com/GafferHQ/gaffer/blob/master/src/GafferUIModule/GadgetBinding.cpp#L221), really. I guess the same applies there?

Reply:
Yeah - looks like the original was wrong too. Could you fix that up?

Reply:
Done.